diff --git a/plugin/surround.vim b/plugin/surround.vim
index 6a986e8..c81acf7 100644
--- a/plugin/surround.vim
+++ b/plugin/surround.vim
@@ -577,8 +577,8 @@ inoremap <silent> <Plug>ISurround  <C-R>=<SID>insert(1)<CR>
 
 if !exists("g:surround_no_mappings") || ! g:surround_no_mappings
   nmap ds  <Plug>Dsurround
-  nmap cs  <Plug>Csurround
-  nmap cS  <Plug>CSurround
+  nmap ss  <Plug>Csurround
+  nmap sS  <Plug>CSurround
   nmap ys  <Plug>Ysurround
   nmap yS  <Plug>YSurround
   nmap yss <Plug>Yssurround
